client/app/bundles/HelloWorld/components/Header.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react';
+
+export default class HelloWorld extends React.Component {
+
+    render() {
+        return (
+            <header className="business-header">
+                <div className="container">
+                    <div className="row">
+                        <div className="col-lg-12  text-center">
+                            <br /><br /><br /><br /><br /><br />
+                                <hr />
+                                    <a href="/">
+                                        <h1 className="tagline">WeEatAgain</h1>
+                                    </a>
+                                    <hr />
```

Comment:
I also have a slight OCD and I totally missed that, should someone take my OCD title away from me? :/
(fixed it)